sympy/integrals/__init__.py

```diff
@@ -9,7 +9,11 @@
     >>> integrate(sin(x),x)
     -cos(x)
 """
+from .deltafunctions import deltaintegrate
+from .heurisch import heurisch, heurisch_wrapper
 from .integrals import integrate, Integral, line_integrate
+from .manualintegrate import manualintegrate, integral_steps
+from .rationaltools import ratint
```

Comment:
Am 09.03.2015 um 14:26 schrieb Sergey B Kirpichev:
>>   from .integrals import integrate, Integral, line_integrate
>> +from .manualintegrate import manualintegrate, integral_steps
>> +from .rationaltools import ratint
>
> On Mon, Mar 09, 2015 at 06:09:10AM -0700, Joachim Durchholz wrote:
>>     As I can see in the stable version, the docs examples are correct, not
>>     wrong:
>
> Yes, because they don't have "shortened" imports.

What do you mean with '"shortened" import'?

> PS: Actually, this all was a note for your reviewer, not for you.

So you'd like me to stay out of the loop, eh?
I'm open to arguments why that might be a good idea, but currently I see 
more reasons to stay in rather than stay out.